app/views/DR/EpidemiologicalResponseScreen/mentalHealthAdvices.js

```diff
@@ -1,9 +1,99 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { View } from 'react-native';
+import PDFView from 'react-native-pdf';
 
-//This screen will be fill up with 'Epidemiological random tips' PR#60
-const MentalHealthAdvices = () => {
-  return <View />;
-};
+import Colors from '../../../constants/colors';
+import { GetStoreData, SetStoreData } from '../../../helpers/General';
+
+class MentalHealthAdvices extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      date: '',
+      tip: '',
+    };
+  }
+
+  getData = async () => {
+    const storageData = JSON.parse(await GetStoreData('epidemiologicalTips'));
```

Comment:
If you send `false` in the second parameter, we don't need to parse the JSON
https://github.com/intellisysdcorp/covid-safe-paths/blob/develop/app/helpers/General.js#L10-L23

```suggestion
    const storageData = await GetStoreData('epidemiologicalTips', false);
```